### PR TITLE
V2.01 (first errata)

### PR DESCRIPTION
**Pull request by andrewbraunstein**
Errata adds corrections missed from first PR.

### CODE REVIEW

**Comment by cdolan** [comment]
I think this needs a rebase on the latest `master` commit.

File: patient-roster.md [hunk 1]
```diff
@@ -162,14 +162,14 @@ These values are not case-sensitive. That is, `female`, `FEMALE`, and `Female` a
 
 #### <a name="ssn-format"></a>SSN format
 
-An SSN may be written with the traditional hyphenation or without.
-So, `123-45-6789` and `123456789` are both allowed.
+A SSN may be written with the traditional hyphenation or without.
```
Comment (97jaz) [inline patient-roster.md]: Wouldn't this be pronounced "ess-ess-en"? If so, I'd expect "an" here. (https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html)

File: patient-roster.md [hunk 2]
```diff
@@ -162,14 +162,14 @@ These values are not case-sensitive. That is, `female`, `FEMALE`, and `Female` a
...
+So, `123-45-6789` and `123456789` are both allowed.  Unknown or not applicable SSN should be omitted.
 
 
 #### <a name="phone-format"></a>Phone format
 
 The preferred (but not required) format for US phone numbers is
-`(123) 456-7890`.
+`(123) 456-7890`.  The Home phone number should be submitted if Cell phone number is submitted, even if they are the same number - i.e. submit the same number in both.
```
Comment (97jaz) [inline patient-roster.md]: I probably wouldn't capitalize "Home" or "Cell" in this sentence, but I can understand why you do. I don't feel strongly about this. I do think the hyphen should be replaced by a proper dash (`—`).

File: patient-roster.md (hunk 1 above)
Comment (andrewbraunstein) [reply patient-roster.md]: Has been changed back to 'An'

File: patient-roster.md (hunk 2 above)
Comment (andrewbraunstein) [reply patient-roster.md]: changed to dash.

File: patient-roster.md [hunk 3]
```diff
@@ -103,11 +103,11 @@ Each record in a patient roster file contains the following fields, in the follo
 3. \* Last name
 4. \* First name
 5. Middle name
-6. Name suffix (e.g., "Jr.," "III", etc.)
+6. Name suffix (e.g., "Jr.", "III", etc.)
```
[the remark this answers is not in the document]
Comment (andrewbraunstein) [reply patient-roster.md]: treating each as a block and quotes removed.

File: patient-roster.md [hunk 4]
```diff
@@ -163,13 +163,13 @@ These values are not case-sensitive. That is, `female`, `FEMALE`, and `Female` a
...
 An SSN may be written with the traditional hyphenation or without.
-So, `123-45-6789` and `123456789` are both allowed.
+So, `123-45-6789` and `123456789` are both allowed.  Unknown or not applicable SSN should be omitted.
 
 
 #### <a name="phone-format"></a>Phone format
 
 The preferred (but not required) format for US phone numbers is
-`(123) 456-7890`.
+`(123) 456-7890`.  The home phone number should be submitted if cell phone number is submitted, even if they are the same number  — i.e. submit the same number in both.
```
Comment (97jaz) [inline patient-roster.md]: There's an extra space to the left of the dash.

**Pull request merged**